alumnium/drivers/appium_driver.py

```diff
@@ -10,6 +10,7 @@
 
 
 from alumnium.accessibility import XCUITestAccessibilityTree
+from alumnium.accessibility.uiautomator2_accessibility_tree import UIAutomator2AccessibiltyTree
```

Comment:
You can add `UIAutomator2AccessibiltyTree` to `__init__.py` so it can be imported from `alumnium.accessibility` directly.